tests/connection_e2e_fcr2port_test.go

```diff
@@ -16,6 +16,6 @@ func TestCloudRouter2PortCreateConnection(t *testing.T) {
 	defer terraform.Destroy(t, terraformOptions)
 
 	terraform.InitAndApply(t, terraformOptions)
-	output := terraform.Output(t, terraformOptions, "fcr2port_result")
+	output := terraform.Output(t, terraformOptions, "fcr2port_connection_result")
```

Comment:
This should remain "fcr2port_result" to match the output in the corresponding script main.tf file.

Reply:
Ack.